packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1003,9 +1003,17 @@ export class CrossChainMessenger {
    * @returns Current challenge period in seconds.
    */
   public async getChallengePeriodSeconds(): Promise<number> {
-    const challengePeriod = this.bedrock
-      ? await this.contracts.l1.L2OutputOracle.FINALIZATION_PERIOD_SECONDS()
-      : await this.contracts.l1.StateCommitmentChain.FRAUD_PROOF_WINDOW()
+    if (!this.bedrock) {
+      return (await this.contracts.l1.StateCommitmentChain.FRAUD_PROOF_WINDOW()).toNumber()
+    }
+
+    const oracle_version = await this.contracts.l1.L2OutputOracle.version()
+    const challengePeriod = oracle_version === '1.0.0'
+      ? BigNumber.from(await this.contracts.l1.OptimismPortal.provider.call({
+        to: this.contracts.l1.OptimismPortal.address,
+        data: '0xf4daa291' // FINALIZATION_PERIOD_SECOND
```

Comment:
```suggestion
        data: '0xf4daa291' // FINALIZATION_PERIOD_SECONDS
```

Reply:
Worth explaining in a comment why this is using a `call` rather than a function. I understand it but someone reading the code might not.